test/conftest.py

```diff
@@ -6,12 +6,6 @@ def pytest_addoption(parser):
                      help="Only run tests marked for Travis")
 
 
-def pytest_runtest_setup(item):
-    not_travis = item.get_marker("not_travis")
-    if not_travis is not None and item.config.getoption("--travis"):
-        pytest.skip("Skipping test marked not for Travis")
```

Comment:
@wence- I guess we can now remove all Travis related bits in `conftest.py`? Do we still need the `still alive` hack?

Reply:
No

Reply:
Okay, I've cleaned up `conftest.py` some more.